config/src/config/mempool_config.rs

```diff
@@ -60,7 +60,9 @@ pub struct MempoolConfig {
     pub broadcast_buckets: Vec<u64>,
     pub eager_expire_threshold_ms: Option<u64>,
     pub eager_expire_time_ms: u64,
-
+    /// Uses the MempoolSyncMessageV2 instead of MempoolSyncMessage when sending mempool transactions
+    /// to upstream nodes.
+    pub use_mempool_sync_message_v2: bool,
```

Comment:
How about we call the new message type `BroadcastTransactionsRequestV2` -> `BroadcastTransactionsRequestWithReadyTime` or something similar that has significance (since we're not going to deprecate "v1"). Maybe this can be `include_ready_time_in_broadcast`?

Reply:
Done.